opnode/node/node.go

```diff
@@ -2,85 +2,175 @@ package node
 
 import (
 	"context"
+	"fmt"
 	"time"
 
+	"github.com/ethereum-optimism/optimistic-specs/opnode/l1"
+	"github.com/ethereum-optimism/optimistic-specs/opnode/l2"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/event"
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum/go-ethereum/ethclient"
+
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
 type OpNodeCmd struct {
-	L1NodeAddr   string `ask:"--l1" help:"Address of L1 User JSON-RPC endpoint to use (eth namespace required)"`
-	L2EngineAddr string `ask:"--l2" help:"Address of L2 Engine JSON-RPC endpoint to use (engine and eth namespace required)"`
+	L1NodeAddrs   []string `ask:"--l1" help:"Addresses of L1 User JSON-RPC endpoints to use (eth namespace required)"`
+	L2EngineAddrs []string `ask:"--l2" help:"Addresses of L2 Engine JSON-RPC endpoints to use (engine and eth namespace required)"`
```

Comment:
Would be great to document the format of JSON-RPC endpoints (what are the engine and eth namespaces?) and giving an example. Can be in markdown doc and not in code.